src/snowflake/snowpark/udf.py

```diff
@@ -66,6 +67,14 @@ def __call__(
         self,
         *cols: Union[ColumnOrName, Iterable[ColumnOrName]],
     ) -> Column:
+        if len(cols) >= 1 and isinstance(cols[0], (list, tuple)):
+            warnings.warn(
```

Comment:
Can we warn only once instead of warning for every call?
`@deprecate` warn only once.

Reply:
changed to use `logging.warning` and created https://snowflakecomputing.atlassian.net/browse/SNOW-659918 for only warning once, which will be done in another PR